govcd/disk_test.go

```diff
@@ -621,3 +608,64 @@ func (vcd *TestVCD) Test_QueryDisk(check *C) {
 	}
 
 }
+
+// Tests Disk retrieval by name, by ID, and by a combination of name and ID
```

Comment:
if we decide to change the behavior of `GetDiskByName`, or to remove it, this testing function will not work anymore and the tests performed by `testFinderGetGenericEntities` should be performed explicitly.

Reply:
test changed